embuilder.py

```diff
@@ -31,7 +31,9 @@
 # Minimal subset of SYSTEM_TASKS used by CI systems to build enough to useful
 MINIMAL_TASKS = [
     'libcompiler_rt',
+    'libcompiler_rt-mt',
     'libc',
+    'libc-mt',
```

Comment:
are these separate fixes, things that were missing before?

Reply:
Ooops you are right.. the real fix is below.  Reverted.